### ISSUE TITLE
feature request: add key combo to jump to specific user-input rooms

### ISSUE THREAD

**Issue by Soundtoxin** [dino/dino#892]
Something like the "go" plugins for irssi and weechat or the built-in C-k bind in Element and Nheko. You hit the key and then a box appears and you start to type a room name, then it filters by name as you type. This lets you quickly jump around when you have a lot of rooms, without the need for a mouse. Currently I think you have to just press ctrl-tab or ctrl-shift-tab a lot if you don't want to use the mouse.

**Comment by selurvedu** [comment]
Sounds exactly like https://github.com/dino/dino/issues/369#issuecomment-502442364

**Comment by Soundtoxin** [comment]
Yes, it does. The issue that comment is on doesn't sound all that similar, though. (in case you were going to close this as a duplicate)

**Comment by selurvedu** [comment]
No, I wasn't, and I don't have the power to do so :upside_down_face:
I left that comment to make github add a notice to # 369 that somebody mentioned it, thus making a reference.
